menus/utils/utils.ts

```diff
@@ -68,42 +58,48 @@ export async function isCtypeOnChain(ctype: ICType): Promise<boolean> {
 }
 export const resolveOn = ChainHelpers.Blockchain.IS_FINALIZED
 
+export async function loadBalance(account: KiltKeyringPair, testnet: any) {
+  const api = ConfigService.get('api')
+
+  let balance = parseInt(
+    (await api.query.system.account(account.address)).data.free.toString()
+  )
+  if (balance < 3) {
+    const message = testnet
+      ? chalk.red(
+          `get testnet tokens to continue => https://faucet.peregrine.kilt.io/?${account.address}`
+        )
```

Comment:
Maybe put orange here? 

Red is more after a problem occurs, like on line 72.

Reply:
```suggestion
    const message = testnet
      ? chalk.orange(
          `get testnet tokens to continue => https://faucet.peregrine.kilt.io/?${account.address}`
        )
```